packages/dds/merge-tree/src/test/obliterate.reconnect.spec.ts

```diff
@@ -209,3 +210,222 @@ for (const { incremental, mergeTreeEnableSidedObliterate } of generatePairwiseOp
+			{
+				removalType: "remove",
+				getRemoveMethod: (helper: ClientTestHelper): ClientTestHelper["removeRange"] =>
+					helper.removeRange.bind(helper),
+			},
+			{
+				removalType: "obliterate",
+				getRemoveMethod: (helper: ClientTestHelper): ClientTestHelper["removeRange"] =>
+					helper.obliterateRange.bind(helper),
+			},
+		]) {
+			itCorrectlyObliterates({
+				title: `${removalType} overlapping obliterate start`,
+				action: (helper) => {
+					helper.insertText("A", 0, "0123456789");
+					helper.processAllOps();
+					helper.disconnect("B");
+					helper.obliterateRange(
+						"B",
+						{ pos: 1, side: Side.Before },
+						{ pos: 8, side: Side.After },
+					);
+					helper.insertText("A", 5, "should be obliterated");
+					getRemoveMethod(helper)("A", 0, 2);
+					helper.reconnect("B");
+				},
+				expectedText: "9",
+			});
+
+			itCorrectlyObliterates({
+				title: `${removalType} overlapping obliterate middle`,
+				action: (helper) => {
+					helper.insertText("A", 0, "0123456789");
+					helper.processAllOps();
+					helper.disconnect("B");
+					helper.obliterateRange(
+						"B",
+						{ pos: 1, side: Side.Before },
+						{ pos: 8, side: Side.After },
+					);
+					getRemoveMethod(helper)("A", 3, 4);
+					helper.insertText("A", 5, "should be obliterated");
+					helper.reconnect("B");
+				},
+				expectedText: "09",
+			});
+
+			itCorrectlyObliterates({
+				title: `${removalType} overlapping obliterate end`,
+				action: (helper) => {
+					helper.insertText("A", 0, "0123456789");
+					helper.processAllOps();
+					helper.disconnect("B");
+					helper.obliterateRange(
+						"B",
+						{ pos: 1, side: Side.Before },
+						{ pos: 8, side: Side.After },
+					);
+					getRemoveMethod(helper)("A", 8, 10);
+					helper.insertText("A", 5, "should be obliterated");
+					helper.reconnect("B");
+				},
+				expectedText: "0",
+			});
+		}
+	});
+
+	// This test and the analogous endpoint test below provide some rationale for the policy of tending to shrink obliterates
+	// inward upon reconnect in the case that regions near their endpoints were removed between the original submission and resubmission.
+	itCorrectlyObliterates({
+		title: "obliterate shrinks start point to best possible option",
+		action: (helper) => {
+			helper.insertText("A", 0, "0123456789");
+			helper.processAllOps();
+			helper.disconnect("B");
+			helper.obliterateRange("B", { pos: 3, side: Side.After }, { pos: 8, side: Side.Before }); // 4 through 7
+			helper.insertText("A", 7, "inside the original obliterate");
+			helper.removeRange("C", 1, 5); // 1234
+			helper.advanceClients("B");
+			// B recognizes an observer client at this seq will have '056inside the original obliterate789' and B's obliterate should additionally
+			// remove '5inside the original obliterate67'.
+			// It reconnects at this point and has a choice: it can either specify the startpoing of the obliterate as 'after the 0 character'
```

Comment:
```suggestion
			// It reconnects at this point and has a choice: it can either specify the startpoint of the obliterate as 'after the 0 character'
```